app/lib/features/categories/organize_categories_page.dart

```diff
@@ -0,0 +1,269 @@
+import 'package:acter/common/providers/room_providers.dart';
+import 'package:acter/common/toolkit/buttons/primary_action_button.dart';
+import 'package:acter/common/widgets/room/room_card.dart';
+import 'package:acter/features/categories/actions/save_categories.dart';
+import 'package:acter/features/categories/model/CategoryModelLocal.dart';
+import 'package:acter/features/categories/providers/categories_providers.dart';
+import 'package:acter/features/categories/utils/category_utils.dart';
+import 'package:acter/features/categories/widgets/add_edit_category.dart';
+import 'package:acter/features/categories/widgets/category_header_view.dart';
+import 'package:drag_and_drop_lists/drag_and_drop_lists.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:phosphor_flutter/phosphor_flutter.dart';
+
+class OrganizeCategoriesPage extends ConsumerStatefulWidget {
```

Comment:
One common screen which do job for all the categories management including add/update/delete category and drag and drop ordering management for list and sub-entries.

Create code in generic way so that it can easy plug in for any module.